.github/workflows/e2e.yaml

```
@@ -0,0 +1,101 @@
+on:
```

```suggestion
name: E2e tests
on:
```
Should give this workflow a nice name

---

Ok, wasn't sure as the one we had for the dashboard was anonymous. Will add the name there.